python-threatexchange/threatexchange/signal_type/pdq_index.py

```diff
@@ -31,15 +31,15 @@ def __init__(self, entries: t.Iterable[t.Tuple[str, T]]) -> None:
     def __len__(self) -> int:
         return len(self.local_id_to_entry)
 
-    def query(self, hash: str) -> t.List[IndexMatch[T]]:
+    def query(
+        self, hash: str, threshhold=PDQ_CONFIDENT_MATCH_THRESHOLD
```

Comment:
(blocking): This no longer matches the interface on the base class @ https://github.com/facebook/ThreatExchange/blob/main/python-threatexchange/threatexchange/signal_type/index.py#L89